vendoredApps

```diff
@@ -0,0 +1,28 @@
+// Copyright 2015 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Vendor in actool, which is used in building the stage1 ACI
+github.com/appc/spec/actool
+
+// Vendor in CNI plugins
```

Comment:
I'd rather suggest having better comments then. Like:

Vendor in CNI plugins, which are built and put in stage1 images.

and

Vendor in ACE, which is used in functional tests